swagger.json

```diff
@@ -4740,12 +4738,6 @@
                             "description": "The time at which the assigned driver is estimated to arrive at the job destination. Only valid for en-route jobs.",
                             "example": 1462881998034
                         },
```

Comment:
Deleting a duplicate instance of `destination_address_id` since `DispatchJob` should pick it up from the `DispatchJobCreate` def

Reply:
I think I just committed a fix for this earlier this morning.

Reply:
Awesome, thanks! That probably explains the merge conflict.